Czaple/Models/FirstLaunch.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  FirstLaunch.swift
+//  Czaple
+//
+//  Created by Filip Trzaska on 05/12/2019.
+//  Copyright © 2019 Filip Trzaska. All rights reserved.
+//
+
+import UIKit
+
+protocol FirstLaunchProtocol {
+    func checkFirstLaunch() -> Bool
```

Comment:
You can change it to simple 
`var isFirstLaunch: Bool { get }`

Reply:
And add method that would set it true to use it when it would be first launch. Better testability, single responsibility :)